index.d.ts

```diff
@@ -0,0 +1,50 @@
+export as namespace EventEmitter;
```

Comment:
It's true that we cannot support both the `import =` and the ES6-style imports with the type definitions. We need to choose one. I put a bit of consideration into this when I wrote the defs. 

The problem with `import =` is that if you use it anywhere in your code, you cannot compile into ES6 modules. I need to enable that setting, which was my motivation for this PR as well as #71.

Any users who are currently using this repository with TypeScript either wrote their own definitions or are using the (old) ones from [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/eventemitter3/eventemitter3.d.ts). I wonder what happens if this repository had these definitions included *and* the user had the DT definitions installed as well. That may allow legacy code to continue working while new projects can use the ES6 import syntax.